packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -68,7 +74,8 @@ const enableKernelGC = true;
 // bundle.$BUNDLEID = JSON(bundle)
 //
 // kernel.defaultManagerType = managerType
-// kernel.defaultReapInterval = $NN
+// (old) kernel.defaultReapInterval = $NN
```

Comment:
Yeah, we're still in that fuzzy ad-hoc stage of this table's maturity curve, but I'll improve the comments and write down the definitions of v0/v1/whatever even though we don't have a single key that says what version the kvStore is at. For swingstore I'm working on making it precise (and doing a full+atomic migration from vN to vN+1), but for this table, the weird thing was that vatKeepers have an "upgrade your keys" function, which the kernel calls at startup (sometimes), so there's a brief window during this call where vat4 has migrated but vat5 has not, and that didn't lend itself to declaring the entire kvStore as at "version 2", and I didn't want to introduce per-vat version keys either.

Reply:
Ok I overhauled the way upgrade happens, and now there *is* a `version` key that indicates how we're using the kvStore. There's an explicit `upgradeSwingset()` that must be called at least once per swingset-package-version -changing reboot (but is safe to call on every reboot), and it look for `version < 1` or a missing `version` to trigger the 0-to-1 upgrade.

Unlike my plans for swing-store SQL schemas, new kernels start out at version 1, partly because there are different kernel-wide options that influence the v1 behavior, which were missing at v0, so I'm not sure the final state would match what I want it to be.

I'll be pushing a new branch soon.